src/screencapturekit.cs

```diff
@@ -187,6 +233,11 @@ interface SCShareableContent {
 
 		[Export ("applications")]
 		SCRunningApplication [] Applications { get; }
+
+		[Mac (14, 0)]
+		[Static]
+		[Export ("infoForFilter:")]
+		SCShareableContentInfo Info (SCContentFilter filter);
```

Comment:
All methods names must begin with a Verb 

```suggestion
		SCShareableContentInfo GetInfo (SCContentFilter filter);
```